lib/rucio/core/rse.py

```diff
@@ -1229,7 +1249,12 @@ def update_rse(rse_id, parameters, session=None):
     :param session: The database session in use.
 
     :raises RSENotFound: If RSE is not found.
+    :raises InputValidationError: If a parameter does not exist. Nothing will be added then.
     """
+    for key in parameters.keys():
+        if key not in MUTABLE_RSE_PROPERTIES:
+            raise exception.InputValidationError(f"The key '{key}' does not exist for RSE properties.")
```

Comment:
I don’t know if the following is better or worse, but I’ve been recently seeing it a lot, so just putting it for your consideration.
```python
if any(key not in MUTABLE_RSE_PROPERTIES for key in parameters.keys()):
    raise exception.InputValidationError(f"The key '{key}' does not exist for RSE properties.")
```

Reply:
I like the idea and changed it. However, the variable `key` is used in the for loop to create a more descriptive error message.

Reply:
We could write something like
```python
    for key in parameters.keys():
        raise exception.InputValidationError(f"The key '{key}' does not exist for RSE properties.") if key not in MUTABLE_RSE_PROPERTIES
```
However, I don't like this syntax. It "hides" the condition and makes the line unnecessary longer, while just saving one line.

Reply:
Darn, you’re very right, I missed this implication. Apologies for the noise.